app/shedular.js

```diff
@@ -106,7 +108,24 @@ var shedular = {
         all_jobs = scheduler.scheduledJobs;
         result = all_jobs[jobName].reschedule(cron_expression);
         return result;
-    }
+    },
+    notifyOpenedTicket: async function () {
+        try {
+            const config = await AppConfig.getConfig();
+            var job = scheduler.scheduleJob(config.notifyOpenedTicket, function () {
+                var config = {
+                    channels: ['Email']
```

Comment:
any config constant Email,Team has to be in single file so if we have to make changes ..we do not need to change in multiple places